Sources/Sentry/Public/SentryMeasurementUnit.h

```diff
@@ -0,0 +1,104 @@
+#import "SentryDefines.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+NS_SWIFT_NAME(MeasurementUnit)
+@interface SentryMeasurementUnit : NSObject <NSCopying>
+SENTRY_NO_INIT
+
+- (instancetype)initWithUnit:(NSString *)unit;
+
+@property (readonly, copy) NSString *unit;
+
+@end
+
+NS_SWIFT_NAME(MeasurementUnitDuration)
+@interface SentryMeasurementUnitDuration : SentryMeasurementUnit
+SENTRY_NO_INIT
+
+/** Nanosecond, 10^-9 seconds. */
+@property (class, readonly, copy) SentryMeasurementUnitDuration *nanosecond;
+
+/** Microsecond , 10^-6 seconds. */
+@property (class, readonly, copy) SentryMeasurementUnitDuration *microsecond;
+
+/** Millisecond, 10^-3 seconds. */
+@property (class, readonly, copy) SentryMeasurementUnitDuration *millisecond;
+
+/** Full second. */
+@property (class, readonly, copy) SentryMeasurementUnitDuration *second;
+
+/** Minute, 60 seconds. */
+@property (class, readonly, copy) SentryMeasurementUnitDuration *minute;
+
+/** Hour, 3600 seconds. */
+@property (class, readonly, copy) SentryMeasurementUnitDuration *hour;
+
+/** Day, 86,400 seconds. */
+@property (class, readonly, copy) SentryMeasurementUnitDuration *day;
+
+/** Week, 604,800 seconds. */
+@property (class, readonly, copy) SentryMeasurementUnitDuration *week;
+
+@end
+
+NS_SWIFT_NAME(MeasurementUnitInformation)
+@interface SentryMeasurementUnitInformation : SentryMeasurementUnit
```

Comment:
l: UnitInformation for me doesn't mean much. I would prefer something like UnitBytes.

Reply:
That aligns with other SDKs. On Apple, it's called `NSUnitInformationStorage`, see https://developer.apple.com/documentation/foundation/nsunitinformationstorage?language=objc